taskt/UI/Forms/frmScriptBuilder.cs

```diff
@@ -1461,71 +1461,71 @@ private void openToolStripMenuItem_Click(object sender, EventArgs e)
 
             //if user selected file
             if (openFileDialog.ShowDialog() == DialogResult.OK)
-            {
-                //open file
-                OpenFile(openFileDialog.FileName);
-            }
-        }
-
-        private void OpenFile(string filePath)
-        {
-            try
-            {
-                //get deserialized script
-                Script deserializedScript = Script.DeserializeFile(filePath);
-
-                //check if script is a part of the currently opened project
-                string openScriptProjectName = deserializedScript.ProjectName;
-
-                if (openScriptProjectName != _scriptProject.GetProjectName())
-                    throw new Exception("Attempted to load a script not part of the currently open project");
-
-                //reinitialize
-                lstScriptActions.Items.Clear();
-                _scriptVariables = new List<ScriptVariable>();
-
-                if (deserializedScript.Commands.Count == 0)
-                {
-                    Notify("Error Parsing File: Commands not found!");
-                }
-
-                //update file path and reflect in title bar
-                ScriptFilePath = filePath;
-
-                //assign variables
-                _scriptVariables.AddRange(deserializedScript.Variables);
-
-                //update ProjectPath variable
-                var projectPathVariable = _scriptVariables.Where(v => v.VariableName == "ProjectPath").SingleOrDefault();
-                if (projectPathVariable != null)
-                    projectPathVariable.VariableValue = _scriptProjectPath;
-                else
-                {
-                    projectPathVariable = new ScriptVariable
-                    {
-                        VariableName = "ProjectPath",
-                        VariableValue = _scriptProjectPath
-                    };
-                    _scriptVariables.Add(projectPathVariable);
-                }
-
-                //populate commands
-                PopulateExecutionCommands(deserializedScript.Commands);
-
-                //notify
-                Notify("Script Loaded Successfully!");
-            }
-            catch (Exception ex)
-            {
-                //signal an error has happened
-                Notify("An Error Occured: " + ex.Message);
-            }
-        }
-
+            {
+                //open file
+                OpenFile(openFileDialog.FileName);
+            }
+        }
+
+        private void OpenFile(string filePath)
+        {
+            try
+            {
+                //get deserialized script
+                Script deserializedScript = Script.DeserializeFile(filePath);
+
+                //check if script is a part of the currently opened project
+                string openScriptProjectName = deserializedScript.ProjectName;
+
+                if (openScriptProjectName != _scriptProject.GetProjectName())
```

Comment:
Long term it may make more sense to simply open it and display a warning that "this file is not part of the current project". Especially if we add tabs to the core UI for open scripts.